scripts/runironic-exporter

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash
 
-# Set dummy provisioning IP to avoid interface detection issues (not needed to run IPE to service `/metrics`)
-export PROVISIONING_IP="127.0.0.1"
+# Skip provisioning interface detection (not needed for exporter)
+export SKIP_PROVISIONING_INTERFACE=true
```

Comment:
I think we need to go down a different path instead of adding yet another undocumented variable. I suggest splitting the provisioning interface logic into a new bash helper and not including it automatically from configure-ironic.

Note that regardless of the implementation, configure-ironic now needs to handle non-existing IRONIC_IP. This is the hardest part, but it's also the case here.